.travis.yml

```diff
@@ -94,20 +87,6 @@ jobs:
         - source "$DEV_LIB_PATH/travis.script.sh"
         - npm run test:js -- --ci --cacheDirectory="$HOME/.jest-cache"
 
-    - name: E2E tests
-      php: "7.3"
-      env: WP_VERSION=latest DEV_LIB_SKIP=phpcs,eslint,xmllint,phpsyntax,phpunit PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=
-      install:
-        - nvm install
-        - composer install
-        - npm install
-      script:
-        - npm run build:js
-        - npm run build:css
-        - npm run env:start
-        - npm run test:e2e:ci
-        - npm run env:stop
-
```

Comment:
I don't think the tests have to be removed as a while. Rather, perhaps the `tests/e2e/specs/stories-editor` directory should be removed.

Reply:
Ah OK.

Reply:
I would not delete the directory.

Instead, I recommend adjusting `testPathIgnorePatterns` in `tests/e2e/jest.config.js` to ignore the `stories-editor` directory.

Reply:
Some tests in `tests/e2e/specs/block-editor` are also failing. Should those be ignored as well?